spec.md

```diff
@@ -4399,6 +4434,8 @@ primop_1expr2int = primop_1expr2int_keyword , "(" , expr , "," , int , "," , int
 (* Primitive Property Operations *)
 property_primop_2expr = property_primop_2expr_keyword ,
                           "(" , property_expr , "," , property_expr ")" ;
+property_primop_varexpr = property_primop_varexpr_keyword ,
+                            "(" , property_expr , [ "," , property_expr ]* ")" ;
```

Comment:
This should support zero arguments (in spec text above and as implemented), but as written requires at least one property argument.

Also, I think `{`s are used for zero-or-more, not kleene star.

Reply:
Thanks, I missed the `{` `}` instead of kleene star. I think this should now indicate zero arguments the way we normally specify zero or more.